src/router.js

```diff
@@ -12,12 +12,28 @@ import DocsSearch from './components/docs/Search.vue';
 
 Vue.use(Router);
 
-export default new Router({
+const router = new Router({
   routes: [
     { path: '/', name: 'home', component: HomePage },
     { path: '/docs', name: 'docs', component: DocumentationPage, children: [
       { path: ':source', name: 'docs-source', component: DocsLoader, children: [
-        { path: ':tag', name: 'docs-tag', component: DocsViewer, children: [
+        { path: ':tag', name: 'docs-tag', component: DocsViewer, meta: {
+          title: route => {
+            const { class: clarse, typedef, file } = route.params;
+            const name = clarse || typedef || file || 'Search';
+            let rest = '';
+            if (name === 'Search') {
+              rest = `: ${route.params.q || ''}`;
```

Comment:
This would result in visiting the search page with no query saying "Search: " - wouldn't it be better to have it just say "Search" in the event of no query?